src/content/vintage/advanced/observability/logging/log-shipping/index.md

```diff
@@ -0,0 +1,39 @@
+---
+linkTitle: How to ship audit logs
+title: How to ship audit logs to a remote location
+description: This article explains how to forward your audit logs to a remote location.
+weight: 50
+menu:
+  main:
+    identifier: advanced-observability-logging-logshipping
+    parent: advanced-observability-logging
+user_questions:
+  - How do I ship audit logs to a remote location?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2024-02-28
+aliases:
+  - /advanced/observability/logging/log-shipping
+---
+
+In this article you will learn how you can forward your audit logs to a remote location (e.g. Security Information and Event Management system).
```

Comment:
```suggestion
In this article you will learn how to forward audit logs to external storage, like a Security Information and Event Management (SIEM) system.
```
Just to help improve searchability, since "remote location" has been used already